thunder/tests/opinfos.py

```diff
@@ -7805,6 +7910,11 @@ def cross_entropy_sample_generator(op, device, dtype, requires_grad, **kwargs):
                 datatypes.bfloat16,
             ),
         ),
+        # TODO FIXME -- These tests are hitting an odd issue where real torch tensors are being passed to nll_loss
```

Comment:
This skip is inherited from `nll_loss` because we're using it in the cross-entropy decomposition.